src/stores/businessStore.ts

```diff
@@ -0,0 +1,290 @@
+import { AllowedActionsIF, ApiBusinessIF, BusinessStateIF, BusinessWarningIF } from '@/interfaces'
+import { defineStore } from 'pinia'
+import { CorpTypeCd, EntityState } from '@/enums'
+import { DateUtilities, LegalServices } from '@/services/'
+import { GetCorpNumberedDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const useBusinessStore = defineStore('business', {
+  state: (): BusinessStateIF => ({
+    businessInfo: {
+      adminFreeze: null,
+      allowedActions: null,
+      associationType: null,
+      foundingDate: null,
+      goodStanding: null,
+      hasCorrections: null,
+      hasCourtOrders: null,
+      hasRestrictions: null,
+      identifier: null,
+      lastAddressChangeDate: null,
+      lastAnnualReportDate: null,
+      lastDirectorChangeDate: null,
+      legalName: null,
+      legalType: null,
+      naicsCode: null,
+      naicsDescription: null,
+      naicsKey: null,
+      nextAnnualReport: null,
+      state: null,
+      stateFiling: null,
+      warnings: []
+    }
+  }),
+
+  getters: {
+    /** The allowed actions object. */
+    getAllowedActions (state): AllowedActionsIF {
```

Comment:
In your IDE, what is the type of `state` when you hover on it here? If it's "any", please specify the actual type ("BusinessStateIF" I think).

That way we get typing on "businessInfo" and "allowedActions" etc below.

Reply:
Great great catch Sev. I tried this out and I can see the type of `businessInfo`and `allowedActions` now 👍

Reply:
Excellent. Are you fixing them all?

Reply:
Yes. I've fixed all of them across all stores.